test/common/index.js

```diff
@@ -618,11 +617,17 @@ function expectsError(fn, settings, exact) {
 }
 
 function skipIfInspectorDisabled() {
-  if (process.config.variables.v8_enable_inspector === 0) {
+  if (!process.features.inspector) {
     skip('V8 inspector is disabled');
   }
 }
 
+function skipIfInspectorEnabled() {
```

Comment:
I'm not sure it's worth increasing the surface area of `common` for this as it's only used in one place?